client/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import allProductsReducer from './allProducts'
+
+export default combineReducers({
+  products: allProductsReducer
+})
+
+export * from './allProducts'
```

Comment:
Do we need to import * from './allProducts' in this file in order to be able to export *?

Reply:
Don't need to explicitly import from './allProducts' in this file. Should work as is.